tools/tx_pull.py

```diff
@@ -107,13 +108,62 @@ def get_parser():
     parser.add_argument(
         '-e', '--email', dest='email',
         help=('Provides an email address used to commit on github if the one '
-              'associated to the github account is not pulbic'))
+              'associated to the GitHub account is not public'))
     parser.add_argument(
         '-t', '--target-org', dest='target',
-        help=('By default, translation are committed in github on OCA. This '
+        help=('By default, translation are committed in GitHub on OCA. This '
               'arg lets you provide an alternative org'))
     return parser
 
+
+def wrap_tx_call(func, args=None, kwargs=None):
+    """Intercept all TX calls to wait when the API rate limit is reached."""
+    while True:
```

Comment:
I'm not entirely sure architecture of how this is run in production, but is this threaded at all? This could cause an infinite loop if Transifex goes down I think, which could cause some stacking of sockets